### PR TITLE
feat(bot): adaptivecards-tools sdk

### PR DESCRIPTION
**Pull request by YitongFeng-git**
alpha version published https://www.npmjs.com/package/@microsoft/adaptivecards-tools

### CODE REVIEW

File: packages/adaptivecards-tools-sdk/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "adaptivecards-tools",
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:2]: better named like "@microsoft/*****"

File: packages/adaptivecards-tools-sdk/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "version": "1.0.0",
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:3]: better start from "0.1.0"

File: packages/adaptivecards-tools-sdk/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "@rollup/plugin-replace": "^2.2.0",
+    "@types/react": "^17.0.14",
+    "@types/markdown-it": "^12.2.1",
+    "rimraf": "^3.0.0",
+    "rollup": "^2.59.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "rollup-plugin-terser": "^5.1.1",
+    "typescript": "^4.4.2"
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:26]: please check these version sync up with other packages' devdeps

File: packages/adaptivecards-tools-sdk/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  "types": "types/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib types",
+    "clean-build": "npm run clean && npm run build",
+    "build": "rollup -c"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "rollup-plugin-typescript2": "^0.25.3",
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:15]: please enable eslint and prettier in this pkg

**Comment by wenytang-ms** [comment]
it's better release an alpha version, and test for templates
then merge to repo or release a stable version

**Comment by wenytang-ms** [comment]
next Monday, dev branch will merge to main, please confirm weather this pkg needs to merge to dev branch

File: packages/adaptivecards-tools-sdk/package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "",
+  "main": "./lib/index.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib types",
+    "clean-build": "npm run clean && npm run build",
+    "build": "rollup -c"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (eriolchan) [inline packages/adaptivecards-tools-sdk/package.json:13]: What's the meaning of "ISC" for license?

File: packages/adaptivecards-tools-sdk/package.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  "devDependencies": {
+    "rollup-plugin-typescript2": "^0.25.3",
+    "@rollup/plugin-commonjs": "^13.0.2",
```
Comment (eriolchan) [inline packages/adaptivecards-tools-sdk/package.json:16]: I'm curious on what's rollup library for dev dependency?

File: packages/adaptivecards-tools-sdk/rollup.config.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import sourcemaps from "rollup-plugin-sourcemaps";
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/rollup.config.js:1]: add Microsoft corp. copyright at header

File: packages/adaptivecards-tools-sdk/src/AdaptiveCard.tsx [hunk 8]
```diff
@@ -0,0 +1,38 @@
+import { AdaptiveCards } from "./AdaptiveCards";
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/src/AdaptiveCard.tsx:1]: copyright

File: packages/adaptivecards-tools-sdk/src/AdaptiveCards.ts [hunk 9]
```diff
@@ -0,0 +1,78 @@
+import { Template } from "adaptivecards-templating";
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/src/AdaptiveCards.ts:1]: copyright

**Comment by wenytang-ms** [comment]
better add .gitignore file to filter node_modules and others

File: packages/adaptivecards-tools-sdk/rollup.config.js [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+        file: pkg.main,
+        format: "cjs",
+        sourcemap: true,
+        name: "AdaptiveCardsTools",
+    },
+    preserveSymlinks: false,
+    plugins: [
+        sourcemaps(),
+        replace({
+            delimiters: ["", ""],
```
Comment (henzhang-ms) [inline packages/adaptivecards-tools-sdk/rollup.config.js:22]: Is `""` duplicated?

File: packages/adaptivecards-tools-sdk/package.json (hunk 6 above)
Comment (YitongFeng-git) [reply packages/adaptivecards-tools-sdk/package.json:16]: for pack the package

File: packages/adaptivecards-tools-sdk/rollup.config.js (hunk 10 above)
Comment (YitongFeng-git) [reply packages/adaptivecards-tools-sdk/rollup.config.js:22]: it is expected

**Comment by YitongFeng-git** [comment]
E2E TEST: npm pkg, no e2e test

**Review by wenytang-ms — DISMISSED** [review]
LGTM the CI/CD Related content

File: packages/adaptivecards-tools-sdk/README.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
+# AdaptiveCards SDK for TypeScript/JavaScript
+
+Adaptivecards-tools sdk aims to reduce developer's effort to render an Adaptive Card in your project, especially for Adaptive Card with Data, provide type safety check.
```
Comment (IvanJobs) [inline packages/adaptivecards-tools-sdk/README.md:3]: Maybe: 
  reduce developers' efforts
  in their projects

**Review by IvanJobs — DISMISSED** [review]
lgtm

**Review by eriolchan — APPROVED** [review]

**Pull request merged**